randovania/game_connection/connector/prime1_remote_connector.py

```diff
@@ -55,6 +59,31 @@ def _asset_id_format(self):
     def multiworld_magic_item(self) -> ItemResourceInfo:
         return self.game.resource_database.get_item(prime_items.MULTIWORLD_ITEM)
 
+    async def get_inventory(self) -> Inventory:
+        """Fetches the inventory represented by the given game memory."""
+
+        memory_ops = await self._memory_op_for_items(
+            [item for item in self.game.resource_database.item if not item.extra.get("exclude_from_remote_connector")]
```

Comment:
other way around, we want the <1000 check, not the exclude remote connector

Reply:
on second thought, you can replace the first two blocks with just a `super` call